extensions/db/src/test/java/org/mule/extension/db/integration/insert/BulkInsertTestCase.java

```diff
@@ -43,6 +51,15 @@ public void bulkInsertWithOverriddenType() throws Exception {
     assertBulkInsert(response.getPayload().getValue());
   }
 
+  @Test
+  public void bulkInsertUnusedParameterType() throws Exception {
+    expectedException.expect(MessagingException.class);
+    expectedException.expectCause(instanceOf(IllegalArgumentException.class));
+    expectedException
+        .expectMessage(is("Query defines parameters unused but they aren't present in the query."));
```

Comment:
seems like i was wrong... You do need the brackets and you do need so add single quotes on each parameter name... Sorry :)